tests/unit/ExtensionRequest.spec.ts

```diff
@@ -2,48 +2,351 @@ import Vue from 'vue'
+        } as AgmExtEvalIF
       },
       vuetify
     })
+  })
 
+  afterEach(() => {
+    wrapper.destroy()
+  })
+
+  it('displays the initial view properly', () => {
+    // Verify component headers
     expect(wrapper.find('.v-card').attributes('id')).toBe('extension-request')
     expect(wrapper.find('header i').attributes('class')).toContain('mdi-calendar-range')
     expect(wrapper.find('header h2').text()).toBe('Extension Request')
 
     const rows = wrapper.findAll('.content .row')
 
+    // Verify isFirstAgm radio group
     expect(rows.at(0).find('.col-sm-3').text()).toBe('Is this the first AGM?')
     expect(rows.at(0).find('.col-sm-9 .v-input--radio-group').exists()).toBe(true)
 
+    // Verify AGM year text field
     expect(rows.at(1).find('.col-sm-3').text()).toBe('AGM Year')
-    expect(rows.at(1).find('.col-sm-9 .v-text-field').exists()).toBe(true)
+    expect(rows.at(1).find('#year-txt').exists()).toBe(true)
+    expect(rows.at(1).find('#year-txt').attributes().disabled).toBeDefined()
 
+    // Verify isPrevExtension radio group
     expect(rows.at(2).find('.col-sm-3').text()).toBe('Has an extension been requested for this AGM year already?')
     expect(rows.at(2).find('.col-sm-9').text())
-      .toContain('Yes - Specify the date the extension expiresDate of extension expiry')
+      .toContain('Yes - Specify the date the extension expires')
     expect(rows.at(2).find('.col-sm-9').text()).toContain('No - this is the first extension request for this AGM')
 
+    // Verify intended date picker
     expect(rows.at(3).find('.col-sm-3').text()).toBe('Intended date this AGM will be held')
     expect(rows.at(3).find('.col-sm-9').findComponent(DatePicker).exists()).toBe(true)
+  })
 
-    wrapper.destroy()
+  it('displays first agm with no previous extension requested', async () => {
+    // Click "Yes" radio button for isFirstAgm radio group
+    const firstAgmRadios = wrapper.find('#first-agm-radio-group').findAll('input[type="radio"]')
+    await firstAgmRadios.at(0).trigger('click')
+    await Vue.nextTick() // wait for DOM to update
+
+    // Click "No" radio button for isPrevExtension radio group
+    const prevExtensionRadios = wrapper.find('#prev-extension-radio-group').findAll('input[type="radio"]')
+    await prevExtensionRadios.at(1).trigger('click')
+    await Vue.nextTick() // wait for DOM to update
+
+    const rows = wrapper.findAll('.content .row')
+
+    // Verify isFirstAgm radio group
+    expect(firstAgmRadios.at(0).attributes('aria-checked')).toBe('true')
+    expect(firstAgmRadios.at(1).attributes('aria-checked')).toBe('false')
+
+    // Verify AGM year text field
+    expect(rows.at(1).find('#year-txt').attributes().disabled).toBeDefined()
+    expect(wrapper.vm.data.agmYear).toBe('2023')
+
+    // Verify isPrevExtension radio group
+    expect(prevExtensionRadios.at(0).attributes('aria-checked')).toBe('false')
+    expect(prevExtensionRadios.at(1).attributes('aria-checked')).toBe('true')
```

Comment:
I think there's a way to directly check if the radio button wrapper is checked but I can't think of it off the top of my head and this is OK. FYI.